services/notification/src/cfnresponse.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright 2016 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.
```

Comment:
NOTE: This file is copy/pasted from https://github.com/awslabs/aws-cloudformation-templates/blob/master/aws/services/CloudFormation/MacrosExamples/StackMetrics/lambda/cfnresponse.py; we want our own because that file is actually _not_ shared by AWS via library. We were using https://github.com/gene1wood/cfnresponse, but we prefer to have full control of the file.

Reply:
Is this the same exact as is in "discovery"? Could we put this one one location to share with ourselves?